bin/init.bash

```diff
@@ -320,6 +319,17 @@ set_nginx_config() {
     fi
 }
 
+set_nginx_config_ingress_override() {
+    file="${1}"
+    if [[ ! -f "${file}" ]]; then
+        log_error "ERROR: invalid file - ${file}"
+        exit 1
+    fi
+  if [ "${CK8S_CLOUD_PROVIDER}" = "azure" ]; then
+        replace_set_me "${file}" '.networkPolicies.ingressNginx.ingressOverride.ips' '["0.0.0.0/0"]'
+    fi
+}
```

Comment:
Consistent indentation.

Reply:
Hmm, for some reason VSCode sometimes flips to 2 spaces instead of 4. I wonder if we have different settings in different projects or files?

Reply:
https://github.com/elastisys/compliantkubernetes-apps/pull/2095/commits/c5a48a8a6a6d418c166fe12fb31e1aa12bb61e94